student1registeredCourse.txt

```diff
@@ -0,0 +1,3 @@
+CPT112 - DISCRETE STRUCTURES
```

Comment:
remove the `student1registeredCourse.txt`, it is created when signing up but is not necessary for the application until a real user use it

Reply:
Done all the correction above